.flake8

```diff
@@ -2,7 +2,7 @@
 select = ANN,A50,B,B9,BLK,C,D,DAR,E,F,I,S,W
 docstring-convention = google
 max-complexity = 10
-ignore = ANN101,B009,E203,E501,W503,DAR101,DAR201,D100,D103,D107,D101,D104
+ignore = ANN101,B009,E203,E501,W503,DAR101,DAR201,D100,D103,D107,D101,D104,BLK100
```

Comment:
Ikke ignorer black, Kjør heller nox sesjonen for black, den fikser alt som feiler på BLK100 automatisk